src/test/java/uk/gov/hmcts/ccd/endpoint/std/GlobalSearchEndpointTest.java

```diff
@@ -0,0 +1,98 @@
+package uk.gov.hmcts.ccd.endpoint.std;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import uk.gov.hmcts.ccd.domain.model.search.global.GlobalSearchRequestPayload;
+import uk.gov.hmcts.ccd.domain.model.search.global.GlobalSearchSortByCategory;
+import uk.gov.hmcts.ccd.domain.model.search.global.GlobalSearchSortDirection;
+import uk.gov.hmcts.ccd.domain.model.search.global.SearchCriteria;
+import uk.gov.hmcts.ccd.domain.model.search.global.SortCriteria;
+import uk.gov.hmcts.ccd.domain.service.globalsearch.GlobalSearchService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class GlobalSearchEndpointTest {
```

Comment:
```suggestion
public class GlobalSearchRequestPayloadTest {
```
The tests in this class aren't testing the endpoint; they're testing the `GlobalSearchRequestPayload` methods so should be moved to the appropriate location